src/components/Icon/types/IconPropsType.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export type IconPropsType = {
+  color?: any;
+  size?: any;
+  direction?: any;
```

Comment:
```suggestion
  color?: ICONS_COLOR;
  size?: ICONS_SIZE;
  direction?: any;
```

Also do we have a enum for the direction?